protocol_driver_grpc.h

```diff
@@ -106,6 +106,8 @@ class ProtocolDriverGrpc : public ProtocolDriver {
   void HandleConnectFailure(std::string_view local_connection_info) override;
 
   std::vector<TransportStat> GetTransportStats() override;
+  std::vector<TransportStat> GetTransportStatsServer() override;
```

Comment:
I don't think these need to exist actually.

Reply:
Removed